cmd/create.go

```diff
@@ -228,10 +231,20 @@ func (cc *createCmd) generateDockerfile(langConfig *config.DraftConfig, lowerLan
 		return errors.New("supported languages were loaded incorrectly")
 	}
 
+	flagVariablesMap := make(map[string]string)
```

Comment:
looks like we have to do this chunk twice, so it could be cleaner to do it once higher in the call stack at the run() or createFiles() methods and pass the flagVariablesMap down to generateDockerfile() and createDeployment() by adding a new parameter? let me know what you think 😄

Reply:
I've made the flagVariablesMap public instead.